include/mapbox/vector_tile/builder.hpp

```diff
@@ -8,6 +8,11 @@
 namespace mapbox {
 namespace vector_tile {
 
+template <typename T>
+vtzero::point_2d convert(mapbox::geometry::point<T> pt) {
```

Comment:
@joto I was required to create this `convert` method now during encoding for it to accept my generic `mapbox::geometry::point<T>` type. Is there a better way to do this then sprinkling my convert as required?

Reply:
At first I had the plan of doing conversion of point types magically behind the scenes when a special function was overloaded (and there is still some documentation that talks about this, I need to fix this). But then I decided to remove that, because it needs a lot of magic that can be hard to understand when not working. Sprinkling the `convert`s everywhere isn't that hard to do and makes it very obvious what is happening here.